wasm2c/wasm-rt.h

```diff
@@ -51,6 +51,13 @@ extern "C" {
 #define wasm_rt_unreachable abort
 #endif
 
+/**
+ * The default WASM page size
```

Comment:
"default" seems odd since this is never configurable.

Maybe you could split out this non-functional change that just renames this macro?

Reply:
Actually, for option 3, this wouldn't be needed, so i think I can revert this change